benchmark/hpp/benchmark_ics.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <iscool/signals/signal.h>
+
+#include "../../benchmark.hpp"
+
+class Ics
+{
+    NOINLINE(void handler(Rng& rng))
+    {
+        volatile std::size_t a = rng(); (void)a;
+    }
+
+    public:
+
+    using Signal = iscool::signals::signal<void(Rng&)>;
+
+    template <typename Subject, typename Foo>
+    static void connect_method(Subject& subject, Foo& foo)
+    {
+        subject.connect(std::bind(&Foo::handler, &foo, std::placeholders::_1));
```

Comment:
This most likely is the location of why iscool::signals isn't passing the validation method. All benchmarked libraries need some form of automatic disconnect functionality whether that be through a base class such as Nano::Observer or a connection object such as in dacap signal with its scoped_connection. (as a side note i still don't know what boost::signals2 doesn't pass it either...)